src/emissions/emission.py

```diff
@@ -0,0 +1,504 @@
+# Emissions class
+import numpy as np
+import tomllib
+
+from AEIC.performance_model import PerformanceModel
+from AEIC.trajectories.trajectory import Trajectory
+from emissions.APU_emissions import get_APU_emissions
+from emissions.EI_CO2 import EI_CO2
```

Comment:
I think we've got something messed up again in the directory structure -- there shouldn't be anything internal to AEIC that needs to import from `AEIC`, nor anything in this module that needs to reference its name explicitly -- these imports should end up being something like `from .performance_model import ...` and `from . APU_emissions import ...`.